LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/ILoggerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan
+{
+    using Microsoft.Extensions.Logging;
+    using System;
```

Comment:
using System should appear before anything else

Reply:
It's ordered alphabetical - if we want to enforce the ordering differently, we should setup the tooling to do so. I was thinking to re-order and have system on top, however, that is limiting the tooling support also for VS, where we can simply cleanup the usings with a single click and they get ordered alphabetically. For productivity, I keep this.

Reply:
I will take care of this Once this damn branch works

Reply:
actually, found the setting that we were looking for and added it to the .editorconfig. With that we have VS support and they get ordered System first alphabetically.